tests/kernel/iop/interrupt/register.expected

```diff
@@ -0,0 +1,10 @@
+-- TEST BEGIN
+multiple register:
+  register when no handler is present: 0
+  register when handler is already present: -104
+  handler result: 43690
```

Comment:
Is this number a vcount or something?  Is it consistent between runs?

-[Unknown]